dropshot/tests/fail/bad_endpoint1.rs

```diff
@@ -1,5 +1,7 @@
 // Copyright 2020 Oxide Computer Company
 
+#![allow(unused_imports)]
```

Comment:
It is. TL;DR: Unused import warnings that are omitted when compiler errors occur are frequently false positives, and in this test suite, they obscure the compiler error we're trying to repro.

Added a comment to this effect.

Reply:
Nice.  Thanks for the comment!

Reply:
Am I reading this correctly that the "unused import" warning only occurred in test 8? Did you change make it so that we would have seen these warnings in more/all tests?

My preference would be that the output of these tests actually reflect what users would see.